src/modules/ShortcutGuide/ShortcutGuide/shortcut_guide.h

```diff
@@ -45,15 +48,15 @@ class OverlayWindow
 
     struct OverlayOpacity
     {
-        PCWSTR name = L"overlay_opacity";
-        int value = 90; // percent
+        static inline PCWSTR name = L"overlay_opacity";
+        int value;
```

Comment:
Why you removed the default value?

Reply:
I moved them to the settings file. Ideally, we should move `DisabledApps`, `OpenShortcut`, `Theme`, `OverlayOpacity` structs there as well but it's not the scope of this PR.

Reply:
OK